config/config.sample.yaml

```diff
@@ -33,6 +33,7 @@ oauth2:
 
```

Comment:
You also need to create a changelog file as per https://github.com/matrix-org/matrix-appservice-slack/blob/develop/CONTRIBUTING.md#making-pull-requests

Reply:
Also, please sign off :)

Reply:
Done!